### ISSUE TITLE
error

### ISSUE THREAD

**Issue by anemul** [EbonJaeger/perworldinventory-kt#123]
**Describe the bug**
Happens on join

**To Reproduce**
1. join the server that is it

**Expected behavior**
not cause the error

**Plugin version**
- [x] Using the latest released version of the plugin

**Server version**
1.13.2

**Additional context and logs**
https://pastebin.com/VPK9q5vL

**Comment by random1167** [comment]
same
```
 Could not pass event PlayerSpawnLocationEvent to PerWorldInventory v2.2.2
org.bukkit.event.EventException: null
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:316) ~[spigot-1.13.2.jar:git-Spigot-cf694d8-5259d80]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:66) ~[spigot-1.13.2.jar:git-Spigot-cf694d8-5259d80]
        at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:507) [spigot-1.13.2.jar:git-Spigot-cf694d8-5259d80]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:492) [spigot-1.13.2.jar:git-Spigot-cf694d8-5259d80]
        at net.minecraft.server.v1_13_R2.PlayerList.a(PlayerList.java:122) [spigot-1.13.2.jar:git-Spigot-cf694d8-5259d80]
        at net.minecraft.server.v1_13_R2.LoginListener.b(LoginListener.java:149) [spigot-1.13.2.jar:git-Spigot-cf694d8-5259d80]
        at net.minecraft.server.v1_13_R2.LoginListener.tick(LoginListener.java:53) [spigot-1.13.2.jar:git-Spigot-cf694d8-5259d80]
        at net.minecraft.server.v1_13_R2.NetworkManager.a(NetworkManager.java:230) [spigot-1.13.2.jar:git-Spigot-cf694d8-5259d80]
        at net.minecraft.server.v1_13_R2.ServerConnection.c(ServerConnection.java:119) [spigot-1.13.2.jar:git-Spigot-cf694d8-5259d80]
        at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:994) [spigot-1.13.2.jar:git-Spigot-cf694d8-5259d80]
        at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:417) [spigot-1.13.2.jar:git-Spigot-cf694d8-5259d80]
        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:831) [spigot-1.13.2.jar:git-Spigot-cf694d8-5259d80]
        at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:729) [spigot-1.13.2.jar:git-Spigot-cf694d8-5259d80]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_201]
Caused by: java.lang.ClassCastException: java.lang.Float cannot be cast to java.lang.Double
        at me.ebonjaeger.perworldinventory.serialization.LocationSerializer.deserialize(LocationSerializer.kt:38) ~[?:?]
        at me.ebonjaeger.perworldinventory.data.FlatFile.getLogout(FlatFile.kt:148) ~[?:?]
        at me.ebonjaeger.perworldinventory.listener.player.PlayerSpawnLocationListener.onPlayerSpawn(PlayerSpawnLocationListener.kt:31) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_201]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_201]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_201]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_201]
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:312) ~[spigot-1.13.2.jar:git-Spigot-cf694d8-5259d80]
        ... 13 more

```

**Comment by EbonJaeger** [comment]
Fixed for 2.3.0 release.
